### ISSUE TITLE
pip install error

### ISSUE THREAD

**Issue by AbrahamSanders** [yxuansu/SimCTG#28]
I am trying to install the `simctg` package to use the evaluation util for the diversity metric, and I get this error. 
The fix seems straightforward - probably just need to replace `sklearn` with `scikit-learn` in requirements.txt.

```
Collecting sklearn
  Downloading sklearn-0.0.post5.tar.gz (3.7 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [18 lines of output]
      The 'sklearn' PyPI package is deprecated, use 'scikit-learn'
      rather than 'sklearn' for pip commands.
      
      Here is how to fix this error in the main use cases:
      - use 'pip install scikit-learn' rather than 'pip install sklearn'
      - replace 'sklearn' by 'scikit-learn' in your pip requirements files
        (requirements.txt, setup.py, setup.cfg, Pipfile, etc ...)
      - if the 'sklearn' package is used by one of your dependencies,
        it would be great if you take some time to track which package uses
        'sklearn' instead of 'scikit-learn' and report it to their issue tracker
      - as a last resort, set the environment variable
        SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True to avoid this error
      
      More information is available at
      https://github.com/scikit-learn/sklearn-pypi-package
      
      If the previous advice does not cover your use case, feel free to report it at
      https://github.com/scikit-learn/sklearn-pypi-package/issues/new
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.
```

**Comment by pphuc25** [comment]
I have the same error, is there any way to fix it?

**Comment by callanwu** [comment]
It seems that maintainers used the deprecated `sklearn` PyPI package in [requirements.txt](https://github.com/yxuansu/SimCTG/blob/main/requirements.txt).
https://github.com/yxuansu/SimCTG/blob/3a91449c8f3133f2c183e481e705b1dbb268589c/requirements.txt#L7
A simple solution is that you can add a prefix `SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True` in your pip command.
